src/styles/Grid.module.scss

```diff
@@ -0,0 +1,123 @@
+.grid-images [class*='#{$prefix}--col'] {
+    @include carbon--breakpoint-down('md') {
+      padding-right: 0px;
+    }
+  }
+
+// gallery clusters
+.mock-gallery {
+    @include carbon--breakpoint-down('md') {
+      --space: 0;
+    }
+  }
+  
+  .mock-gallery [class*='#{$prefix}--col'] {
+    @include carbon--breakpoint-down('md') {
+      margin-top: 1.5rem;
+    }
+  }
+
+// grid classnames (should be added to either grid package or theme?)
+.#{$prefix}--no-gutter-lg {
+  @include carbon--breakpoint('lg') {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+.#{$prefix}--no-gutter-md {
+  @include carbon--breakpoint('md') {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+.#{$prefix}--no-gutter-sm {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.#{$prefix}--no-gutter-lg--left {
+  @include carbon--breakpoint('lg') {
+    padding-left: 0;
+  }
+}
+
+.#{$prefix}--no-gutter-md--left {
+  @include carbon--breakpoint('md') {
+    padding-left: 0;
+  }
+}
+
+.#{$prefix}--no-gutter-sm--left {
+  padding-left: 0;
+}
+
+//---------------------------------------
+// Overrides
+//---------------------------------------
+
+// Principles/Layout page
+.principles-section [class*='#{$prefix}--col'],
+.example-section [class*='#{$prefix}--col'] {
+  margin-bottom: 1.5rem;
+  @include carbon--breakpoint('md') {
+    margin-bottom: 0;
+  }
+}
+
+.example-section {
+  padding-top: $layout-06;
+}
+
+.example-section:before {
+  content: '';
+  display: block;
+  background: $ui-04;
+  height: 1px;
+  width: calc(100% - 2rem);
+  margin: -$spacing-07 0 0 $spacing-05;
+}
+
+.example-section h3 {
+  margin-top: 0;
+}
+
+.example-aside {
+  margin-top: $spacing-04;
+}
+
+.example-aside:before {
+  width: $spacing-07 !important;
+  left: 0 !important;;
+}
+
+.bx--aspect-ratio--image {
+  background: $carbon--white-0;
+}
+
+.bx--aspect-ratio--image .gatsby-resp-image-wrapper {
```

Comment:
if possible we want to avoid styling this, If an image is added without gatsby-image (e.g. imported and used as `src`) it won't have this class.

Reply:
Let me know if you're not able to get that workin!